src/SprykerEco/Client/Unzer/Dependency/Client/UnzerToZedRequestClientBridge.php

```diff
@@ -31,7 +31,7 @@ public function __construct($zedRequestClient)
      *
      * @return \Spryker\Shared\Kernel\Transfer\TransferInterface
      */
-    public function call($url, TransferInterface $object, $requestOptions = null)
+    public function call($url, TransferInterface $object, $requestOptions = null): TransferInterface
```

Comment:
This should be reverted. Same goes for interface change as well

Reply:
Fixed